assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/buttons/PriceModifierActions.tsx

```diff
@@ -11,17 +11,12 @@ const style: CSSProperties = {
 };
 
 const PriceModifierActions: React.FC<PriceModifierProps> = ({ index, price }) => {
-	const actions = [];
-
-	actions.push(<AddPriceModifierButton key={'add'} index={index} />);
-
-	if (!price.isBasePrice) {
-		actions.push(<DeletePriceModifierButton key={'delete'} price={price} />);
-	}
 	return (
-		<div className='modifier-actions' style={style}>
-			{actions}
+		<div className='ee-price-modifier-actions' style={style}>
+			<AddPriceModifierButton key={'add'} index={index} />
+			{!price.isBasePrice && <DeletePriceModifierButton key={'delete'} price={price} />}
```

Comment:
Nice change 👍

Reply:
that `actions` array was a bit of overengineering